contracts/interfaces/IAEX9Minimal.aes

```diff
@@ -7,6 +7,16 @@ contract interface IAEX9Minimal =
 
     record meta_info = { name : string , symbol : string , decimals : int }
 
+    datatype event
```

Comment:
I'd just add this in the ACI, not in the interface itself

Reply:
In tests at least, we are compiling the router with all the other dependent contracts (DexFactory, DexPairs, and interfaces, including IAEX9Minimal.aes), after the compilation we deploy the contract, and as a result, we get the contract instance we are using it for tests. Is there a place for injecting some extra ACI, in this process? 
Also, those events are part of  IAEX9 standard